src/it/java/se/knowit/bookitevent/ClientUtilIT.java

```diff
@@ -0,0 +1,59 @@
+package se.knowit.bookitevent;
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.jupiter.api.Test;
+
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.MediaType;
+import org.springframework.util.Assert;
+import org.springframework.web.client.RestTemplate;
+
+import se.knowit.bookitevent.model.Event;
+
+public class ClientUtilIT {
+	
+	@Test
+	public void getEventByIdDemo() {
+		HttpHeaders headers = new HttpHeaders();
+		headers.setContentType(MediaType.APPLICATION_JSON);
+		RestTemplate restTemplate = new RestTemplate();
+		String url = "http://localhost:8080/api/v1/events/{id}";
+		
+		Map<String, String> par = new HashMap<>();
+        par.put("id", "72ab7c8b-c0d5-4ab2-8c63-5cf1ad0b439b");
+
+		Event eve = restTemplate.getForObject(url, Event.class, par);
+		System.out.println("Id:" + eve.getId() + " : " + eve.getName() + " : " + eve.getDescription());
+		Assert.hasText(Long.toString(eve.getId()),"1");
+		Assert.hasText(eve.getDescription(), "Ett himla bra event!");
+	}
+
+	@Test
+	public void getAllEventsDemo() {
```

Comment:
namn bör börja på test

Reply:
There are many opinions on names of  tests :-)
https://dzone.com/articles/7-popular-unit-test-naming